Inbound/0x006B.py

```diff
@@ -50,14 +50,14 @@ def error_handler(code): # sub_9EF4B0
         add_long("OwnerAccountId")
     error_handler(result)
 elif f == 10: # confirm place cube
-    error = add_bool("error")
+    error = add_byte("Send Only Player ObjectId") # Also error code
+    add_int("OwnerObjectId")
     add_int("UserObjectId")
-    if error: # This is sent first, item is removed from warehouse
-        add_int("UserObjectId")
+    if error == 44 or error == 1: # If 44: Can't place here
```

Comment:
why only 44 and 1? anything except 0 should be an error?

Reply:
So 1, still works and its meant to only send Owner object id and user object id, the only other I found was 44 which is "Can't place here"

Reply:
fixed it, should be better now

Reply:
This is the code:
![image](https://user-images.githubusercontent.com/7710340/165450213-b883de52-b98f-4c2c-ad32-1e1c7a52ca58.png)

`sub_BEE060` checks if some value is == 2

I have no idea how 44 comes into play here, but I guess...

Reply:
Interesting 🤔

Reply:
Up to you if you wanna change or not then. just lmk what you want

Reply:
it's fine, knowing 44 means something is at least a hint even if that's not exactly what it is.